flexmeasures/api/v3_0/tests/test_sensors_api.py

```diff
@@ -128,29 +173,33 @@ def test_fetch_sensors(
     )
 
     print("Server responded with:\n%s" % response.json)
-    assert response.status_code == 200
 
-    if use_pagination:
-        assert isinstance(response.json["data"][0], dict)
-        assert is_valid_unit(response.json["data"][0]["unit"])
-        assert response.json["num-records"] == exp_num_results
-        assert response.json["filtered-records"] == exp_num_results
+    if test_errors:
```

Comment:
This would be much more to the point if you used `expected_status_code` which is `200` most of the time, but sometimes `403`.